build.gradle

```diff
@@ -71,6 +71,7 @@ allprojects {
 
         testCompile "junit:junit:$junitVersion"
         testCompile "org.loadui:testFx:$testFxVersion"
+        compile files('src/main/resources/jkeymaster-1.1.jar')
```

Comment:
Could we use this instead?
```groovy
compile fileTree(dir: 'lib', include: '*.jar')
```
The jar would go in lib.

I searched a bit more and jars aren't really considered resources, so it would be a better idea to have a separate directory for them.